CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## v0.9.0 (TBD)
 
-
+- Added a check for batch and block proofs to be greater than the minimum security level (#780).
```

Comment:
This doesn't affect the user(s) and is a no-op, so no need for a changelog item.

```suggestion
```